<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_mahdi</class>
 <widget class="QMainWindow" name="gestion_mahdi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>731</width>
      <height>441</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>terrain</string>
     </attribute>
     <widget class="QTableView" name="tableau_terrain">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>71</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLineEdit" name="rech_terrain">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sel_col_terrain">
      <property name="geometry">
       <rect>
        <x>561</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lieu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Etat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Capacite</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lieu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_terrain">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Lieu</string>
      </property>
     </widget>
     <widget class="QComboBox" name="etat">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>            --</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type 4</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="capacite">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>253</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajou_t">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modification_t">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suppression_t">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>reclamation</string>
     </attribute>
     <widget class="QTableView" name="tableau_reclamation">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>481</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLineEdit" name="rech_reclamation">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sel_col_reclamation">
      <property name="geometry">
       <rect>
        <x>561</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Etat</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_reclamation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QComboBox" name="etat_reclamation">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Non traitee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Traitee</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajout_r">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modification_r">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suppression_r">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>suppression</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mail</string>
     </attribute>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>587</x>
        <y>79</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>116</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>522</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>your mail password</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>651</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>menu principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionajout_terrain">
   <property name="text">
    <string>ajout_terrain</string>
   </property>
  </action>
  <action name="actionsuppression_terrain">
   <property name="text">
    <string>suppression_terrain</string>
   </property>
  </action>
  <action name="actionajout_reclamation">
   <property name="text">
    <string>ajout_reclamation</string>
   </property>
  </action>
  <action name="actionsuppression_reclamation">
   <property name="text">
    <string>suppression_reclamation</string>
   </property>
  </action>
  <action name="actionmodifier_terrain">
   <property name="text">
    <string>modifier_terrain</string>
   </property>
  </action>
  <action name="actionmodifier_reclamation">
   <property name="text">
    <string>modifier_reclamation</string>
   </property>
  </action>
  <action name="actioncheck_map">
   <property name="text">
    <string>check_map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
